AxonDeepSeg/apply_model.py

```diff
@@ -1,469 +1,473 @@
 # -*- coding: utf-8 -*-
 
+from pathlib import Path
+import imageio
+
 from scipy.misc import imread, imsave
 from skimage.transform import rescale, resize
+
 from AxonDeepSeg.network_construction import *
-from .config_tools import update_config, default_configuration
-import os
-import imageio
-from .patch_management_tools import im2patches_overlap, patches2im_overlap
-from .visualization.get_masks import get_masks
 import AxonDeepSeg.ads_utils
 
-def apply_convnet(path_acquisitions, acquisitions_resolutions, path_model_folder, config_dict, ckpt_name='model',
-				  inference_batch_size=1, overlap_value=25, resampled_resolutions=[0.1],
-				  prediction_proba_activate=False, gpu_per=1.0, verbosity_level=0):
-	"""
-	Preprocesses the images, transform them into patches, applies the network, stitches the predictions and return them.
-	:param path_acquisitions: List of path to the acquisitions.
-	:param acquisitions_resolutions: List of the acquisitions resolutions (floats).
-	:param path_model_folder: Path to the model folder.
-	:param config_dict: Dictionary containing the model's parameters.
-	:param ckpt_name: String, checkpoint to use.
-	:param inference_batch_size: Int, batch size to use when doing inference.
-	:param overlap_value: Int, number of pixels to use when overlapping the predictions of the network.
-	:param resampled_resolutions: List of resolutions (flaots) to resample to before performing inference.
-	:param prediction_proba_activate: Boolean, whether to compute the probability maps or not.
-	:param gpu_per: Float, percentage of GPU to use if we use it.
-	:param verbosity_level: Int, how much information to display.
-	:return: List of segmentations, and list of probability maps if requested.
-	"""
-
-	# We set the logging from python and Tensorflow to a high level, to avoid messages
-	# in the console when performing segmentation.
-	from logging import ERROR
-	tf.logging.set_verbosity(ERROR)
-	import warnings
-	warnings.filterwarnings('ignore')
-
-	# Network Parameters
-	patch_size = config_dict["trainingset_patchsize"]
-	n_classes = config_dict["n_classes"]
-
-	# STEP 1: Load and rescale the acquisitions, and transform them into patches.
+from .visualization.get_masks import get_masks
+from .patch_management_tools import im2patches_overlap, patches2im_overlap
+from .config_tools import update_config, default_configuration
 
-	rs_acquisitions, rs_coeffs, original_acquisitions_shapes = load_acquisitions(
-		path_acquisitions, acquisitions_resolutions, resampled_resolutions, verbose_mode=verbosity_level)
+def apply_convnet(path_acquisitions, acquisitions_resolutions, path_model_folder, config_dict, ckpt_name='model',
```

Comment:
@maf88 your script that auto-formats styles seems to have been applied to this file, changing every single line. It makes it difficult for me to see what lines you actually changed content (re: paths) vs. formatting (e.g. spaces/tabs) - is there any way you can reverse that? 

Also please turn off any such automated scripts on your computer for the future - if you want to change formatting, a branch should be made and it should be done to all the files at ones, that PR would be easy to review and apply if all that's done was a formatting script. That script should be acknowledged and shared in the wiki too, so that devs don't have competing style formatting scripts.